src/util.ts

```diff
@@ -30,6 +30,23 @@ export async function waitForCursorUpdatesToHappen(): Promise<void> {
   });
 }
 
+/**
+ * FOR TESTING PURPOSES ONLY
```

Comment:
if you want you can add a guard to use if Globals.isTesting is set to true, not a big deal